libs/blog/shared/ui-ad-banner/jest.config.ts

```diff
@@ -0,0 +1,22 @@
+/* eslint-disable */
+export default {
+  displayName: 'ad-banner',
```

Comment:
Wrong name. Change to `blog-shared-ui-ad-banner`

Reply:
Done